src/components/PostDetails.tsx

```diff
@@ -133,9 +81,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     className="delete is-small"
                     aria-label="delete"
                     onClick={() => deleteComment(comment.id)}
-                  >
-                    delete button
-                  </button>
+                  ></button>
```

Comment:
The delete button has an `onClick` event handler that calls `deleteComment` with the comment's id. This is a good practice, but consider adding some visual feedback (like disabling the button) while the deletion is in progress. This can help prevent multiple click events from being fired if the user clicks the button multiple times.